tests/lib/modules.js

```diff
@@ -1,148 +0,0 @@
-/**
```

Comment:
Really that was @caridy's original idea. Just took me a while to see that he was right. :)